static/css/style.css

```diff
@@ -136,6 +136,7 @@ img {
 .our-mission-header-img {
     max-width: 100%;
     width: 100% !important;
+    max-height: 50vh;
```

Comment:
This kills the aspect ratio of the images


![screen shot 2017-05-22 at 2 59 49 pm](https://cloud.githubusercontent.com/assets/4019054/26324304/41a9ca86-3f00-11e7-8aa2-4c20a02f8b47.png)
![screen shot 2017-05-22 at 2 59 39 pm](https://cloud.githubusercontent.com/assets/4019054/26324303/41a8f71e-3f00-11e7-8faf-b1fa05ba9d85.png)